db/models/aggregate_functions.py

```diff
@@ -272,63 +286,60 @@ def get_report_for_all_applications(application_id=None):
             ApplicationsMethods.get_application_by_id(application_id)
         ]
     else:
-        applications = ApplicationsMethods.get_all()
-
+        applications = ApplicationsMethods.get_all(status=Status.SUBMITTED)
     return_json_list = []
     for application in applications:
-        return_json = {
-            "application_id": application.as_dict().get("id"),
-            "asset_type": None,
-            "capital": None,
-            "geography": None,
-            "organisation_type": None,
-            "revenue": None,
-        }
+        return_json = {field: None for field in KEY_REPORT_FIELDS}
         stored_forms = [form.as_json() for form in application.forms]
-        list_of_forms = [
+        report_fields = [
```

Comment:
Im just wondering the diff between KEY_REPORT_FIELDS and this List, is the reason they are separate that they may diverge at some point?

Reply:
Yes, one is the headers for the report and this is the field mapping. I think we could link them but we'd need to extract the mapping out so we can derive the headers. I'll have a go at that.

Reply:
@Adam-W1 OK I've made that change in [9c301d5](https://github.com/communitiesuk/funding-service-design-application-store/pull/96/commits/9c301d5f1357fe647974c831b6fbfb2ff4470f40) if you can re-review.